ui/src/views/Rules/Rule/index.jsx

```diff
@@ -401,6 +448,19 @@ function Rule({ isNewRule, user, ...props }) {
       {error && <ErrorPanel fixed error={error} />}
       {!isLoading && (
         <Fragment>
+          <div>
+            <p className={classes.signoffLabel}>
+              Required Signoffs:
+              <span
+                className={
+                  requiredSignoffs.data &&
```

Comment:
I don't think we need any conditions here. Even when no signoffs are required, it's good to state that explicitly.

Reply:
@bhearsum Not sure what you mean by stating explicitly when no signoffs are required. Does it just mean that I should remove the conditions?

Reply:
That's right. We want to show a string here regardless of whether or not signoffs are required.

Reply:
The conditions are for implementing CSS through the className to grey out the font for the signoffSummary if it is 'Nobody'. But to have the signoffSummary font in black when there is a list of required signoffs.

The signoffSummary string is always shown, it doesn't depend on the conditions.

So, considering this can I leave the conditions as is?

Reply:
It looks like you've mostly got this handled already? I do see that it now says `Nobody` when needed (great!). Are you sure you need the `class` to be conditional? It seems like the class should always be `classes.signoffs` since we always be showing this?